src/mount/readdata_cache.h

```diff
@@ -412,8 +422,11 @@ class ReadCache {
 			reserved_entries_.push_back(*e);
 		} else {
 			assert(e->refcount == 0);
-			std::unique_lock lock(gUsedReadCacheMemoryMutex);
+			std::unique_lock lock(gReadCacheMemoryMutex);
 			gUsedReadCacheMemory -= e->buffer.size();
+			gReadCacheMemoryAlmostExceeded =
+			    gUsedReadCacheMemory >=
+			    static_cast<uint64_t>(0.8 * gReadCacheMaxSize.load());
```

Comment:
Use the already mentioned constant here.

Reply:
Done